### ISSUE TITLE
Contact request

### ISSUE THREAD

**Issue by xme** [eg5846/misp-docker#2]
Hello,
Could you please contact me? I've a question about your MISP container (@xme)

**Comment by eg5846** [comment]
Hello,
you may ask the question, but it could be maybe difficult for me to answer.
I have not used the MISP container (and also MISP) for a long time. I do not maintain or update this project anymore, sorry.

**Comment by xme** [comment]
That’s the purpose of my message.
I built a new image running the latest MISP version based on yours and I’d like to know if I can re-use your initial Dockerfile
(with a mention for you of course)

KR,
Xavier

--
"If the enemy leaves a door open, you must rush in." - Sun Tzu
PGP Key: http://pgp.mit.edu:11371/pks/lookup?op=get&search=0x42D006FD51AD7F2C

**Comment by eg5846** [comment]
Yes, you can re-use my initial Dockerfile. The code is all open. I am happy if you do.
